src/js-based-plugin/index.mjs

```diff
@@ -0,0 +1,39 @@
+import { createUnplugin } from "unplugin";
+import { build } from "ezno";
+
+function emitDiagnostics(on, diagnostics, plugin) {
+	const lineSplits = [0];
+	for (const line of on.split("\n")) {
+		// TODO need buffer length not character count
```

Comment:
lol https://manishearth.github.io/blog/2017/01/14/stop-ascribing-meaning-to-unicode-code-points/